src/bucket/FutureBucket.cpp

```diff
@@ -253,9 +256,28 @@ FutureBucket::getOutputHash() const
     return mOutputBucketHash;
 }
 
+static double
```

Comment:
Maybe use a std::chrono time here rather than double-which-is-a-count-of-something?

Reply:
also I guess we need to have access to this as a metric so that we can compute the ratio in reports?

Reply:
Both done.